templates/kubelet-config.yaml

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
This should be a default file instead of a template. The default file in the role can be overwritten from the playbook.